sw/lib/crt0.S

```diff
@@ -111,6 +89,40 @@ _fp_init:
     // Set FS state to "Clean"
     csrrc x0, mstatus, t1
 
+// Pause all harts except for hart 0 until a IPI is received.
+// On wake-up every core resumes execution from the beginning of main().
+_smp_pause:
+    // Pause harts with hart ID != 0
+    csrr t0, mhartid
+    bnez t0, _wait_for_ipi
```

Comment:
The bootrom and runtime are anyways independent of each other, so for me it would not be a big issue to have it hardcoded to hart zero in one of them. Would be glad to add it as part of this PR if you think it would be valuable!

Reply:
I'm fine with keeping it like this and change it later if there's an actual use case